vocabs/DID-core-v1.jsonld

```diff
@@ -0,0 +1,159 @@
+{
+  "@context": {
+    "range": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#range",
+      "@type": "@id"
+    },
+    "comment": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#comment"
+    },
+    "isDefinedBy": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#isDefinedBy",
+      "@type": "@id"
+    },
+    "sameAs": {
+      "@id": "http://www.w3.org/2002/07/owl#sameAs",
+      "@type": "@id"
+    },
+    "domain": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#domain",
+      "@type": "@id"
+    },
+    "label": {
+      "@id": "http://www.w3.org/2000/01/rdf-schema#label"
+    },
+    "sec": "https://w3id.org/security#",
+    "as": "https://www.w3.org/ns/activitystreams",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "did": "https://www.w3.org/ns/did/vocab/v1#"
+  },
+  "@graph": [
+    {
+      "@id": "https://www.w3.org/ns/did/vocab/v1#",
+      "@type": "owl:Ontology"
+    },
+    {
+      "@id": "did:DIDSubject",
+      "@type": "rdfs:Class",
+      "comment": "Top level class for resources identified via a DID subject",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#core-properties",
+      "label": "DID Document"
+    },
+    {
+      "@id": "did:Service",
+      "@type": "rdfs:Class",
+      "comment": "Service endpoints are used in DID documents to express ways of communicating with the DID subject or associated entities",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#service-endpoints",
+      "label": "Service Endpoint"
+    },
+    {
+      "@id": "did:VerificationMethod",
+      "@type": "rdfs:Class",
+      "comment": "Superclass for all specific verification Methods. It is an 'abstract' class, never used by itself.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#verification-methods",
+      "label": "Verification Method"
+    },
+    {
+      "@id": "did:alsoKnownAs",
+      "@type": "rdf:Property",
+      "comment": "This relationship is a statement that the subject of this identifier is also identified by one or more other identifiers.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#also-known-as",
+      "sameAs": "as:alsoKnownAs"
+    },
+    {
+      "@id": "did:assertionMethod",
+      "@type": "rdf:Property",
+      "comment": "The assertionMethod property is used to express a verification relationship which indicates that a verification method can be used to verify a proof that a statement was asserted on behalf of the DID subject.",
+      "isDefinedBy": "https://www.w3.org/TR/did-core/#assertionmethod",
```

Comment:
```suggestion
      "isDefinedBy": "https://w3id.org/security#assertionMethod",
```

Reply:
I cannot really agree with this (and similar). `assertionMethod` is defined by the normative DID Specification, and not by the Security Vocabulary CG draft report (which, actually, labels this is 'unstable'). Surely the Rec has a priority over the draft report...

Reply:
I am so glad this is finally being said.... when it comes to signatures, they change if this changes... which is why w3id.org was created, so that it could redirect to the appropriate normative definition....

https://w3id.org/security -> https://www.w3.org/TR/did-core/ 

^ this won't work because did core does not define all the terms that security needs and they are identified by fragment, so we cannot redirect there for 1/2 and somewhere else for the other, (like we could for schema.org definitions).

Reply:
> Surely the Rec has a priority over the draft report.

We could say that... and if we say that, we will invalidate all Linked Data Security implementations, all Verifiable Credentials issued to date, and all DID Method testnets and production deployments. :)

As Orie mentions above -- we can't redefine what the ecosystem is doing today w/o asking all implementers to make breaking changes to all of their implementations (and the only gain there will be is design purity, but at the cost of breaking the already deployed ecosystem).

In short, I think it's going to be a very difficult thing to change the URL at this point.

Reply:
??? Are we talking about the same thing?

All the terms and URI-s are now in the original, `sec:` vocabulary. The _only_ thing I am questioning here is whether the `rdfs:isDefinedBy` in the RDFS Vocabulary definition should point at the DID Core spec or the Security Draft. Surely none of the signature methods use this `rdfs:isDefinedBy`, which did not even exist (for DID) a month ago!